mandatory/4-tax.js

```diff
@@ -17,7 +20,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(currency) {
```

Comment:
Using the hint in the comment _(hint: you already wrote a function for this!)_, can you think of another way to calculate the tax in the addTaxAndFormatCurrency function? Remember, functions are written to be reusable and to prevent us from rewriting logic